app/scripts/containers/PackageConfigurePage.jsx

```diff
@@ -19,21 +19,27 @@ export class PackageConfigurePage extends React.Component {
     this.props.initData()
   }
 
+  showLoader = ({ i18n, quote, sitePageSettings }) =>
+    ((sitePageSettings < 3) && !quote)
```

Comment:
maybe it would be better to move all logic here:
`((sitePageSettings < 3) && !quote) || !i18n`
if not then remove unused var i18n